site/book/03-packages/04-updating-a-package.md

```diff
@@ -0,0 +1,48 @@
+Since a forked package contains a reference to its upstream package, kpt can fetch any update to the
```

Comment:
Not only the upstream package, but the exact commit at which point the local fork and the upstream package diverged (i.e. the common ancestor).